docs/getting-started/interact-with-the-cli.md

```diff
@@ -105,3 +105,91 @@ fraud_training                 quickstart
 ```
 
 Listed below are the metadata on that variant, as well as its source and the training sets it belongs to.
+
+## LIST Command
+
+The **LIST** command displays the name, variant and status of all the resources of that resource type. 
+
+```
+featureform list RESOURCE_TYPE --host $FEATUREFORM_HOST –cert $FEATUREFORM_CERT
+```
+
+**RESOURCE\_TYPE** (required) can be:
+
+* **feature -** machine learning features
+* **label -** machine learning labels
+* **training-set -** set of features and one label for training ML models
+* **user -** registered users in your instance
+* **entity -** identifier for a source of features or labels (akin to a primary key)
+* **model -** registered machine learning models which training sets and features are fed to
+* **provider -** registered 3rd party providers which store your data
+* **source -** files, tables, or transformations that features, labels and training sets source from
+
+### Example: Getting the list of users
+
+```
+featureform list users --host $FEATUREFORM_HOST --cert $FEATUREFORM_CERT
+featureform list users --insecure --host $FEATUREFORM_HOST
+```
+
+The commands are both valid ways to retrieve a list of users. The first is with certification. The second without; the --insecure flag disables the need for the --cert flag
```

Comment:
So also, if they use a public cert they don't need the --cert flag at all. The cert flag is for use with self-signed certs